ecosystem/typescript/sdk/.npmignore

```diff
@@ -2,3 +2,4 @@ coverage
 node_modules
 .aptos
 .env
+examples/
```

Comment:
nit: I notice all the other dirs don't have a trailing slash, not sure if that matters